### DISCUSSION TITLE
How change the color of fold char as it appears in the screenshots

### DISCUSSION THREAD

**Discussion by smit2909** [olimorris/onedarkpro.nvim#205] [Q&A]
The screenshots shows the fold character (>) arrow appears yellow for folded region and grey for unfolded region, how to achieve that?

**Comment by olimorris** [comment]
That's from the [status column](https://github.com/olimorris/dotfiles/blob/ad543ac354dcda4051776079d99fc47ce5f40a09/.config/nvim/lua/Oli/plugins/heirline/statuscolumn.lua#L132) in my dotfiles
